library/checkout/checkout.php

```diff
@@ -347,22 +347,61 @@ public function getPhone($address)
         return $phone;
     }
 
-    protected function prepareWrappingArticle($wrappingVat)
+    public function getArticles()
     {
-        $wrappingCost = $this->cart->getOrderTotal(true, CartCore::ONLY_WRAPPING);
-        if ($wrappingCost <= 0) {
-            return [];
+        $products = $this->prepareProductArticles();
+
+        // Get the Tax Rule Group for Wrapping
+        $wrappingTaxRulesGroupId = (int)Configuration::get('PS_GIFT_WRAPPING_TAX_RULES_GROUP');
+        // Get the VAT Rate for the Tax Rule Group
+        $address = new Address($this->cart->id_address_delivery);
+        $tax_manager = TaxManagerFactory::getManager($address, $wrappingTaxRulesGroupId);
+        $tax_calculator = $tax_manager->getTaxCalculator();
+        $wrappingVatRate = $tax_calculator->getTotalRate();
```

Comment:
Not sure what these lines do, can you explain them to me? `PS_GIFT_WRAPPING_TAX_RULES_GROUP` by the name seems to be related to gift wrapping not products, not sure if this is correct